src/abis/Schnaps.ts

```diff
@@ -0,0 +1,171 @@
+export const SchnappsAbi = [
```

Comment:
nit. I'd use default export there as it's the only export that makes sense there.